### DISCUSSION TITLE
Error: 500: //developers.google.com/identity/sign-in/web/devconsole-project. Raw server response: "{"error":{"code":401,"message":"

### DISCUSSION THREAD

**Discussion by WEARNXT** [AntlerVC/firetable#1038] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**

**Comment by BeeBombshell** [comment, marked as answer]
Hi @WEARNXT, it'd be great if you could provide more information on the steps that let you to this error message. Any screenshots or a video recording would be helpful.

**Reply by BeeBombshell** [reply]
Hey @WEARNXT, closing this discussion since it has been unattended for a long time. If you're still facing this issue, feel free to create a new one.
